### DISCUSSION TITLE
Snowflake Procedure Code is not showing up

### DISCUSSION THREAD

**Discussion by harishupadrasta** [open-metadata/OpenMetadata#15955] [General]
Hello, 

We are able to successfully connect to snowflake and sync the data Objects. But we can only see the Procedure names, but the code section is empty for all the procedures.

**Comment by harshach** [comment]
@pmbrull can you check this

**Reply by pmbrull** [reply]
hi @harishupadrasta, you can check the requirements of the Snowflake ingestion [here](https://docs.open-metadata.org/v1.3.x/connectors/database/snowflake)

<img width="1379" alt="image" src="https://github.com/open-metadata/OpenMetadata/assets/35870520/605bb6c5-cd5e-43c6-9804-cf6b7426e718">

Hope this helps

**Comment by harishupadrasta** [comment]
Thank you @pmbrull. Marking this as closed.
